.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: generic
 
+install:
```

Comment:
@faustinoaq We want to use Docker to avoid the tests failing if the crystal version changes.  Also docker-compose provides the postgres db.  Are you thinking we should go away from Docker?

If we decide to use Docker, remove this `install` section.  We already install the shards on the Docker image.